mix.exs

```diff
@@ -49,4 +51,15 @@ defmodule OpencensusElixir.MixProject do
       test: "test --no-start"
     ]
   end
+
+  defp description() do
+    "Elixir library for OpenCensus tracing"
+  end
+
+  defp package() do
+    [
+      licenses: ["Apache 2.0"],
+      links: %{"GitHub" => "https://github.com/opencensus-beam/opencensus_elixir"}
```

Comment:
I would add links to the original library and the organisation, similarly to https://github.com/opencensus-beam/opencensus_plug/blob/ff9d06e7442a421da0ec33b4a121e30826adf16c/mix.exs#L38